index.js

```diff
@@ -62,7 +120,192 @@ function createTaskCreatedAt(taskCreatedAt) {
 function createTaskCheckBox() {
   return `<input
             type="checkbox"
-            class="table-box-check-box"
             name="checkbox-done"
           /> `;
 }
+
+function sortTasks(typeOfSort) {
```

Comment:
`sortType` can be a better variable name.

Reply:
Ok

Reply:
yes but i had same name in outer function